bool visible
bool up
geometry_msgs/Vector3 position
